Sources/SwiftProtobuf/JSONEncoder.swift

```diff
@@ -94,6 +94,12 @@ internal struct JSONEncoder {
         data.append(contentsOf: name.utf8Buffer)
     }
 
+    internal mutating func appendQuoted(name: _NameMap.Name) {
```

Comment:
Done.  Curiously, the `append(name:)` just above is now unused, so I removed that and slid the doc comment down to apply to `appendQuoted(name:)`